docs/reference/announcements-release-notes/880/whats-new-in-88.md

```diff
@@ -54,6 +54,10 @@ Important changes introduced in Camunda 8.8 are summarized as follows:
     <td><p>Identity management is now split into two scopes for improved access management, performance, and flexible integration with any OIDC-compatible Identity Provider:</p><p><ul><li><p>**Identity**: Manages authentication and fine-grained authorizations for the Orchestration Cluster and its APIs.</p></li>
             <li><p>**Management Identity**: Controls access for Web Modeler, Console and Optimize.</p></li></ul></p></td>
 </tr>
+<tr>
+    <td>[Process instance tags](#process-instance-tags)</td>
+    <td>Add immutable, lightweight tags (max 10) at process instance creation for routing, correlation, and prioritization; forwarded to every job and provided in responses.</td>
+</tr>
```

Comment:
💭 @eppdot looking at this page again, I think we should remove Tags from this page completely - this page describes bigger platform changes and does not focus on separate features